examples/src/lib.rs

```diff
@@ -73,28 +77,53 @@ impl ExampleOptions {
             3 => FieldExtension::Cubic,
             val => panic!("'{}' is not a valid field extension option", val),
         };
+
         let hash_fn = match self.hash_fn.as_str() {
             "blake3_192" => HashFunction::Blake3_192,
             "blake3_256" => HashFunction::Blake3_256,
             "sha3_256" => HashFunction::Sha3_256,
+            "rp62_248" => HashFunction::Rp62_248,
+            "rp64_256" => HashFunction::Rp64_256,
             val => panic!("'{}' is not a valid hash function option", val),
         };
 
-        ProofOptions::new(
-            num_queries,
-            blowup_factor,
-            self.grinding_factor,
+        (
+            ProofOptions::new(
+                num_queries,
+                blowup_factor,
+                self.grinding_factor,
+                field_extension,
+                self.folding_factor,
+                256,
+            ),
             hash_fn,
-            field_extension,
-            self.folding_factor,
-            256,
         )
     }
+
+    /// Returns security level of the input proof in bits.
+    pub fn get_security_level(&self, proof: &StarkProof) -> usize {
+        let security_level = match self.hash_fn.as_str() {
+            "blake3_192" => proof.security_level::<Blake3_192>(true),
+            "blake3_256" => proof.security_level::<Blake3_256>(true),
+            "sha3_256" => proof.security_level::<Sha3_256>(true),
+            "rp62_248" => proof.security_level::<Rp62_248>(true),
+            "rp64_256" => proof.security_level::<Rp64_256>(true),
+            val => panic!("'{}' is not a valid hash function option", val),
+        };
+
+        security_level as usize
+    }
 }
 
 #[derive(StructOpt, Debug)]
 //#[structopt(about = "available examples")]
 pub enum ExampleType {
+    /// Compute a Fibonacci sequence using trace table with 2 registers in `f64` basefield.
+    FibSmall {
+        /// Length of Fibonacci sequence; must be a power of two
+        #[structopt(short = "n", default_value = "67072")]
+        sequence_length: usize,
+    },
```

Comment:
nit: let's move this after `Mulfib8`. Also, let's say just "field" instead of "basefield".